rows.go

```diff
@@ -49,9 +48,11 @@ func (fd fieldDesc) Name() string {
 func (fd fieldDesc) Length() (length int64, ok bool) {
 	switch fd.OID {
 	case oid.T_text, oid.T_bytea:
-		return math.MaxInt64, true
-	case oid.T_varchar, oid.T_bpchar:
+		return 4096, true
```

Comment:
@abhishekjog why 4096?

Reply:
TEXT is internal data type with variable length up to 65535. Length of this data type cannot be fetched. If length goes beyond 65535, table is toast. We can display it as -1 as seen in _t_type in catalog.